src/main/java/training/Employees.java

```diff
@@ -0,0 +1,44 @@
+package training;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+public class Employees {

```

Comment:
This class breaks the SOLID principles. Can you identify which one and how to fix it . Please read this article for your help. https://www.baeldung.com/solid-principles